packages/dialog/src/LionDialog.js

```diff
@@ -0,0 +1,42 @@
+import { withModalDialogConfig, OverlayMixin } from '@lion/overlays';
+import { LitElement, html } from '@lion/core';
+
+export class LionDialog extends OverlayMixin(LitElement) {
+  constructor() {
+    super();
+    this.closeEventName = 'dialog-close';
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  _defineOverlayConfig() {
+    return {
+      ...withModalDialogConfig(),
+    };
+  }
+
+  _setupOpenCloseListeners() {
+    this.__close = () => {
```

Comment:
not used anywhere? => remove

Reply:
Was discussing this with Thijs. I meant to make it a protected method to be overridden by extensions. But I think our conclusions after discussing was indeed that this is redundant :)